app/models/student_identifier_type.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class StudentIdentifierType < ActiveRecord::Base
+  default_scope { where(deleted_at: nil) }
+
+  belongs_to :organization, -> { unscope(where: :deleted_at) }
+
+  enum content_type: [:text, :number, :email]
```

Comment:
Hi @tarebyte,

Actually `content_type` here is made to specify the input form type on the assignment invitation page:

![screen shot 2016-06-08 at 10 43 12 pm](https://cloud.githubusercontent.com/assets/1311594/15917410/733da150-2dca-11e6-8aca-e6a311a8b06b.png)

For example, if the teacher set `content_type` to `number`, the identifier form appeared on the assignment invitation page would have type `number`, so that students can not input anything else other than decimal numbers:

![screen shot 2016-06-08 at 10 38 11 pm](https://cloud.githubusercontent.com/assets/1311594/15917344/d0c62f00-2dc9-11e6-95f4-07d6175e88ba.png)

If we change this to `[:name, :id, :email]`, then it may be a little bit confusing with the `name` field. WDYT? 😄